docs/index.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Overview of Pi-hole
+description: Brief overview of Pi-hole
+last_updated: Sun Jan 13 18:22:38 2019 UTC
```

Comment:
Is it possible to automate this via git hooks?

Reply:
It probably is with a pre-commit hook, I didn't get that deep in to it, just pulled up a UTC clock and copy pasted it.